services/core-api/app/api/now_applications/resources/now_application_export_resource.py

```diff
@@ -185,20 +185,47 @@
 ]
```

Comment:
I should get the same formatter as you and/or figure out why mine isn't working. (considering I was the last to update this file)